### ISSUE TITLE
Cannot find module '@influxdata/influxdb3-client' or its corresponding type declarations on 0.3.0

### ISSUE THREAD

**Issue by brjakab** [InfluxCommunity/influxdb3-js#133]
### Specifications

* Client Version: 0.3.0
* InfluxDB Version: 3
* Platform: Cloud serverless

### Code sample to reproduce problem

```ts
import { InfluxDBClient } from '@influxdata/influxdb3-client'
```

### Expected behavior

Imports the package

### Actual behavior

Ts error: Cannot find module '@influxdata/influxdb3-client' or its corresponding type declarations.

### Additional info

Installed using yarn.
0.2.0 works, in node_modules the package does have a dist folder, in 0.3.0 it doesn't have one.

**Comment by bednar** [comment]
Hi @brjakab,

thanks for using client. 

> in node_modules the package does have a dist folder

You are right, we will release fixed version ASAP, meanwhile can you use `0.4.0-nightly.2295` version by: `yarn add influxdata/influxdb3-client@0.4.0-nightly.2295` .


Best

**Comment by bednar** [comment]
The fixed version `3.1` is released.
